xbmc/interfaces/builtins/AddonBuiltins.cpp

```diff
@@ -43,6 +44,30 @@
 
 using namespace ADDON;
 
+/*! \brief Install an addon.
+ *  \param params The parameters.
+ *  \details params[0] = add-on id.
+ */
+static int InstallAddon(const std::vector<std::string>& params)
+{
+  if (params.size())
+  {
+    const std::string& addonid = params[0];
+
+    AddonPtr addon;
+    if (!CAddonMgr::GetInstance().GetAddon(addonid, addon))
+    {
+      CAddonInstaller::GetInstance().InstallModal(addonid, addon);
+    }
+    else
+      CLog::Log(LOGERROR, "InstallAddon called with an addonid of an installed addon.");
```

Comment:
I don't think this should be handled as an error, maybe notice would be enough? If we even have that log level?